temoa/_internal/run_actions.py

```
@@ -203,100 +202,78 @@ def solve_instance(
         try:
-            # currently, the highs solver call will puke if the suffixes are passed, so we need to
-            # differentiate...
+            # currently, the highs solver call will puke if the suffixes are passed
             if solver_name == 'appsi_highs':
                 result = optimizer.solve(instance)
             else:
-                result = optimizer.solve(instance, suffixes=solver_suffixes)
+                result = optimizer.solve(instance, suffixes=solver_suffixes_list)
         except RuntimeError as error:
-            logger.error('Solver failed to solve and returned an error: %s', error)
+            logger.exception('Solver failed to solve and returned an error: %s', error)
```

Ruff (0.14.7)

Redundant exception object included in `logging.exception` call

(TRY401)